jupyterlab/tests/mock_packages/test-hyphens-underscore/pyproject.toml

```
@@ -2,18 +2,66 @@
 # Distributed under the terms of the Modified BSD License.
 
 [build-system]
-requires = ["jupyter_packaging~=0.10,<2", "jupyterlab~=4.0"]
-build-backend = "jupyter_packaging.build_api"
+requires = [
+    "hatchling",
+    "jupyterlab~=4.0",
+]
+build-backend = "hatchling.build"
 
-[tool.jupyter-packaging.options]
-ensured-targets = ["test_hyphens_underscore/labextension/package.json"]
+[project]
+name = "test-hyphens-underscore"
+license = ""
```

```suggestion
license = "BSD-3-Clause"
```

The CI is failing due to missing license:

```
        File "/private/var/folders/2s/h6hvv9ps03xgz_krkkstvq_r0000gn/T/pip-build-env-c2fvbpcd/overlay/lib/python3.12/site-packages/hatchling/metadata/core.py", line 683, in license
          raise ValueError(message) from None
      ValueError: Error parsing field `project.license` - Invalid license expression: ''x
```